tests/static_role_permissions/test_static_role_permissions.py

```diff
@@ -0,0 +1,110 @@
+from unittest import mock
+
+from django.contrib.auth.models import AnonymousUser, User
+from django.test import TestCase, override_settings
+
+from ambient_toolbox.static_role_permissions.permissions import load_static_role_permissions
+
+
+@override_settings(
+    AUTHENTICATION_BACKENDS=[
+        "ambient_toolbox.static_role_permissions.auth_backend.StaticRolePermissionBackend",
+    ]
+)
+class StatiRolePermissionTests(TestCase):
+    @override_settings(STATIC_ROLE_PERMISSIONS_PATH="tests.static_role_permissions.dummy_permissions.PERMISSIONS_DICT")
+    def test_load_permissions_from_settings_path(self):
+        permissions = load_static_role_permissions()
+        self.assertEqual(
+            {"role_1": {"auth.add_user", "auth.view_user"}, "role_2": {}},
+            permissions,
+        )
+
+    def test_load_permissions_from_settings_path_without_path_set(self):
+        with self.assertRaisesMessage(AssertionError, "STATIC_ROLE_PERMISSIONS_PATH is not set in settings.py"):
+            load_static_role_permissions()
+
+    @override_settings(STATIC_ROLE_PERMISSIONS_PATH="tests.static_role_permissions.dummy_permissions.PERMISSIONS_LIST")
+    def test_load_permissions_from_settings_path_with_invalid_dict(self):
+        with self.assertRaisesMessage(AssertionError, "STATIC_ROLE_PERMISSIONS_PATH must point to a dict"):
+            load_static_role_permissions()
+
+    def test_has_perm_without_user_role(self):
+        user = User()  # user.role does not exist
+        self.assertEqual(False, user.has_perm("auth.add_user"))
```

Comment:
question: is this wanted behaviour? to just say "nope, no permissions" when every user has to have a role? 🤔

Reply:
what behaviour would you expect?

The user has no role --> "sure, permission granted" ?

Reply:
haha, i expected maybe an error? but I guess you've thought that through